src/Utilities/Compiler/Analyzer.Utilities.projitems

```diff
@@ -11,6 +11,9 @@
   <PropertyGroup Condition="'$(MicrosoftCodeAnalysisVersion)' == '' OR '$(MicrosoftCodeAnalysisVersion)' &gt;= '3.3'">
     <DefineConstants>$(DefineConstants),CODEANALYSIS_V3_OR_BETTER</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(MicrosoftCodeAnalysisVersion)' == '' OR '$(MicrosoftCodeAnalysisVersion)' &gt;= '3.7'">
+    <DefineConstants>$(DefineConstants),CODEANALYSIS_V3_7_OR_BETTER</DefineConstants>
```

Comment:
I am wondering that now that we don't support FxCopAnalyzers, would it be worth just repurposing `CODEANALYSIS_V3_OR_BETTER` define above to mean `3.7` and also bump up the underlying `Microsoft.CodeAnalysis` version used by the repo to `3.7`? What do you think? https://github.com/dotnet/roslyn-analyzers/blob/690be1702a6ae8f915da3bf6cc1b935e3ff264dd/eng/Versions.props#L28?

Reply:
No objection, but perhaps better as a follow-up?

Reply:
Sounds good.

Reply:
I am still confused though, when is this condition true? None of our existing packages in this repo set `MicrosoftCodeAnalysisVersion` to 3.7 or greater.

Reply:
Roslyn.Diagnostics.Analyzers sets it.

Reply:
Ah, thanks :-)